.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: true
+
+language: bash
+
+dist: trusty
+
+services:
+  - docker
+
+env:
+  - TMP_IMAGE_VER=build-$TRAVIS_BUILD_NUMBER
+
+install:
+  - make clean
+  - make deps dev=yes
+  - make build dev=no
+  - sudo rm -rf node_modules/
+  - make deps dev=no
+  - make dist
+  - make docker.image no-cache=yes VERSION=$TMP_IMAGE_VER
+  - make docker.tar VERSION=$TMP_IMAGE_VER
```

Comment:
@yak80, are`dist.zip` and `release` using anywhere? If no, they also could be removed?